modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/option/CommerceOptionValueHelperImpl.java

```diff
@@ -143,6 +144,15 @@ public List<CommerceOptionValue> getCPDefinitionCommerceOptionValues(
 	public CommerceOptionValue toCommerceOptionValue(String json)
 		throws JSONException {
 
+		if (CPJSONUtil.isEmpty(json)) {
+			return _toCommerceOptionValue(
+				_jsonFactory.createJSONObject(StringPool.BLANK));
+		}
+
+		if (JSONUtil.isJSONArray(json)) {
+			json = json.substring(1, json.length() - 1);
```

Comment:
This is due to us using a JSONArray to parse the json when we validate it. I'm not sure if there's a preferred method of doing this that we have since I couldn't find anything similar to this functionality, but please let me know if we have a Util for it and I'll switch to it.